src/models/__tests__/profile.test.ts

```diff
@@ -112,6 +112,50 @@ describe("findLastVersionByModelId", () => {
     }
   );
 
+  it.each`
```

Comment:
We should add even a test for the other model methods like create and upsert.

Reply:
done in https://github.com/pagopa/io-functions-commons/pull/297/commits/e0d12ae28c6416710e4e67905d9ef48523160e9e